packages/roosterjs-content-model-core/test/utils/convertInlineCssTest.ts

```diff
@@ -120,3 +120,46 @@ describe('convertInlineCss', () => {
         );
     });
 });
+
+describe('splitSelectors', () => {
+    function splitSelectors(selectorText: string) {
```

Comment:
hey I just see that you copied this function into test code. That means the test code is testing test code.

Please do not copy dev code into test code since that means we are not really testing the dev code.

You can export the function from dev code with a comment "for test only", and mark it with `@internal` so you can test it from test code